proto/sentry_protos/snuba/v1/endpoint_get_traces.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+message TraceColumn {
+  enum Name {
```

Comment:
just a few things:
- rename some of these, I would move `enum Name` to top-level scope and call it something like `enum TraceMetadataType` or `TraceKey` or `TraceSelect`
- (subjective) rename Trace in response to just be Row
- (subjective) the whole request might be better called 'TraceTableRequest' or something like that
- add docstrings to at least the top-level messages, and each enum member

Reply:
Why move the enum to top-level? It's really meant to be used only in this endpoint for a request, why not keep it embedded?